src/website/views/components/picker-group.pug

```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
       </label>
     </div>
 
+h3 Texts and icons
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom"&gt;&lt;/i&gt;</code>.
```

Comment:
@janibolkvadze please remove this description as I think it is not necessary. @mattnickles thoughts?